department-of-veterans-affairs/caseflow
# Anya/2954 hearing worksheet part3

connects #2954 

Steps to validate:
1. Go to `<url>/hearings/dockets.json` in the browser
2. Expand `hearing_hash`
3. Ensure `issues` key is present and all fields mentioned in the ticket above are present.

Note: Vacols values such as `program, type, level, note` will be nil until this ticket is done: #3044 

![image](https://user-images.githubusercontent.com/3811786/29616265-e5b107d0-87de-11e7-85c8-ca7e83e30dd9.png)
